foundation-ui/utils/ReactUtil.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ReactUil = {
+  /**
+   * Wrap React elements
+   *
+   * @param {Array.<ReactElement>|ReactElement} elements
+   * @param  {function|String} wrapper component
+   * @param {boolean} [alwaysWrap]
+   *
+   * @returns {ReactElement} wrapped react elements
+   */
+  wrapElements(elements, wrapper = 'div', alwaysWrap = false) {
```

Comment:
Ok, I'm not going to introduce the wrapper props for now as the latter works today, and we can extend the util later if we see need. Hope you guys are ok with that.